test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -435,6 +436,56 @@ public async Task DotnetListPackage_ProjectWithInitialTargets_Succeeds()
             }
         }
 
+        [PlatformTheory(Platform.Windows, Skip = "Enabled after .NET CLI integration for https://github.com/NuGet/Home/issues/9600")]
```

Comment:
I'd create the dotnet/sdk issue. 
Then create a NuGet side issue that basically says, once this PR has been inserted + SDK changes has been merged, re-enable these tests. 

Basically skipped tests should be pointing to an open issue.

Reply:
I talked to William and he said it was fine to link to the NuGet issue on the CLI side instead of having a dedicated dotnet/sdk issue. Let me know if that's not okay.
For re-enabling the tests, I added: https://github.com/NuGet/Home/issues/9800.